charts/neon-storage-controller/templates/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
should storage-controller be available from outside (internet or other VPCs) ? Otherwise we don't need ingress resource at all

Reply:
The network requirements are the same as the storage broker --  it needs to be accessible from outside the kubernetes cluster, but not outside the VPC.

It looks like the broker has this file but also has ingress.enabled=false in all the places we use it.  So we could probably remove it for both broker and controller charts .

@arssher it looks like you originally added this for the broker -- did you have plans for it?
``` 
    * Add ingress template and the values along with it. Disabled by default,
    as it's pretty common practice.
```

Reply:
Since I don't think I need this here, I've removed it from this PR.  We can remove it from the broker separately if we want to.

Reply:
I think I succumbed here to @lassizci advice on common practices; no, I don't have such plans, it doesn't make sense. + 1 for removal.

Reply:
ah, got it, controller should be accessible from pagservers/safekeeprs. Then ingress needed. Or Service type should be LoadBalancer (and use separate AWS ALB/NLB) for that. But internal ingress should be enough.

Reply:
the same for storage broker - we can use ingress instead of separate LB or leave it as is.

Reply:
I think the way it's done for the storage broker is with a LoadBalancer (in the values.yaml files in `aws` repo) -- I've inherited the same way of doing it for controller as well, so hopefully it works.

The load balancer might be overkill: maybe it will make sense to refactor this later for both broker and controller, but I'd like to keep them working the same way for simplicity.